build.zig

```diff
@@ -135,14 +133,14 @@ pub fn build(b: *std.Build) void {
         .PA_BUILDDIR = "/home/pulseaudio/build",
         .PA_CFLAGS = "Not yet supported on meson",
         .PA_DEFAULT_CONFIG_DIR = "/usr/local/etc/pulse",
-        .PA_DLSEARCHPATH = "/usr/local/lib/pulse-14.2/modules",
+        .PA_DLSEARCHPATH = "/usr/local/lib/pulse-16.1/modules",
```

Comment:
On my host this and other paths were very different, so I manually changed the version.